cypress/e2e/debug_clickingInTableOpensReport.cy.js

```diff
@@ -1,37 +1,37 @@
-describe('Clicking a report', function () {
+describe("Clicking a report", function () {
```

Comment:
Not a fan of using single quotes?

Reply:
Linter dingetje

Reply:
It's fine as long as it's consistent. I saw that Martijn also used double quotes.